tools/ci/appveyor/appveyor_build.sh

```diff
@@ -62,7 +62,7 @@ function set_configuration {
             man=''
         ;;
         mingw64)
-            build='--build=i686-pc-cygwin'
+            build='--build=x86_64-pc-cygwin'
```

Comment:
The others could be updated, too - although tbh there's no reason to be passing `--build` these days... perhaps just delete the build variable completely?